op-e2e/actions/reorg_test.go

```diff
@@ -19,14 +20,21 @@ import (
 	"github.com/ethereum-optimism/optimism/op-node/testlog"
 )
 
-func setupReorgTest(t Testing) (*e2eutils.SetupData, *L1Miner, *L2Sequencer, *L2Engine, *L2Verifier, *L2Engine, *L2Batcher) {
+func setupReorgTest(t Testing, fastBlocks bool) (*e2eutils.SetupData, *e2eutils.DeployParams, *L1Miner, *L2Sequencer, *L2Engine, *L2Verifier, *L2Engine, *L2Batcher) {
 	dp := e2eutils.MakeDeployParams(t, defaultRollupTestParams)
```

Comment:
You should pass an instance of `e2eutils.TestParams` here rather than the `fastBlocks` boolean & returning the `e2eutils.DeployParams`. This is getting spaghetti like to pass booleans around like that. You'll need to add the L1 block time to the test params (or can manually create L1 blocks with the 4s block time in the test), but it already has the sequence window size.

Other usages of this function will just pass in `defaultRollupTestParams`

Reply:
Good suggestion, done 👍